doc/classes/BaseButton.xml

```diff
@@ -48,7 +48,7 @@
 		</member>
 		<member name="button_mask" type="int" setter="set_button_mask" getter="get_button_mask">
 			Binary mask to choose which mouse buttons this button will respond to.
-			To allow both left-click and right-click, set this to 3, because it's BUTTON_MASK_LEFT | BUTTON_MASK_RIGHT.
```

Comment:
I was surprised when I found this. We shouldn't be hard-coding the integral value. Even less when we also include the valid code next to it.